### PR TITLE
[SDPA-2620] Block Quote is missing from content template in Body Conten…

### PR DESCRIPTION
**Pull request by vincent-gao**
more details see https://github.com/dpc-sdp/content-vic-gov-au/pull/591

### CODE REVIEW

File: tide_core.install [hunk 1]
```diff
@@ -212,3 +212,18 @@ function tide_core_update_8005() {
...
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_publication') . '/config/install'];
+  if ($wysiwyg_template = Drupal::entityTypeManager()
+    ->getStorage('wysiwyg_template')
+    ->load('blockquote')) {
+    if (count($wysiwyg_template->getNodeTypes()) > 0) {
+      _tide_import_single_config('wysiwyg_template.wysiwyg_template.blockquote', $config_location, TRUE);
+    }
+  }
+}
```
Comment (tim-yao) [inline tide_core.install]: Hey @vincent-gao Missing new line in the end.

File: tide_core.install [hunk 2]
```diff
@@ -212,3 +212,18 @@ function tide_core_update_8005() {
   $config_location = [drupal_get_path('module', 'tide_core') . '/config/optional'];
   _tide_import_single_config('prlp.settings', $config_location);
 }
+
+/**
+ * Enable the blockquote template available to all Node types.
+ */
+function tide_core_update_8006() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_publication') . '/config/install'];
```
Comment (tim-yao) [inline tide_core.install]: @vincent-gao @sonnykt As my understanding, this issue impacted on all content type except `landing_page` and `page`. So should we fix it for `news`, `grants` and all content types in this PR as well?

Comment (sonnykt) [inline tide_core.install]: @vincent-gao  why does the contrib `tide_core` import a config from `tide_publication` which is a `content-vic` only module?

Comment (sonnykt) [reply tide_core.install]: @tim-yao the above code is not correct.

Comment (vincent-gao) [reply tide_core.install]: @sonnykt @tim-yao I will move this code snippet to the publication module. 
this module will keep the `config/install/wysiwyg_template.wysiwyg_template.blockquote.yml`
and the update hook will be moved to `publication` module

**Comment by vincent-gao** [comment]
Hi @anthony-malkoun remove the patch of drupal/entity_embed according to your requirement.

File: tide_core.install [hunk 3]
```diff
@@ -212,3 +212,19 @@ function tide_core_update_8005() {
...
+ * Enable the blockquote template available to all Node types.
+ */
+function tide_core_update_8006() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_core') . '/config/install'];
+  if ($wysiwyg_template = Drupal::entityTypeManager()
+    ->getStorage('wysiwyg_template')
+    ->load('blockquote')) {
+    if (count($wysiwyg_template->getNodeTypes()) > 0) {
+      _tide_import_single_config('wysiwyg_template.wysiwyg_template.blockquote', $config_location, TRUE);
```
Comment (sonnykt) [inline tide_core.install]: @vincent-gao you can always call `$wysiwyg_template->set('node_types', [])->save();` instead of re-importing the config.

Comment (vincent-gao) [reply tide_core.install]: @sonnykt ok thanks.

**Comment by vincent-gao** [comment]
this is really weird, this commit get passed in my local....

File: tide_core.install (hunk 2 above)
Comment (vincent-gao) [reply tide_core.install]: thanks sonny.

**Pull request merged**